providers/google/docs/operators/cloud/dataflow.rst

```diff
@@ -232,6 +232,15 @@ source, such as Pub/Sub, in your pipeline (for Java).
     :start-after: [START howto_operator_start_streaming_python_job]
     :end-before: [END howto_operator_start_streaming_python_job]
 
+Deferrable mode:
+
+.. exampleinclude:: /../../providers/google/tests/system/google/cloud/dataflow/example_dataflow_streaming_python.py
```

Comment:
Could we also update documentation for streaming pipelines with deferrable mode for Java and add a System Test? Or is it better to do it in another PR?